src/api/endpoints/redemption_requests.py

```diff
@@ -366,3 +375,23 @@ def put(self, redeem_id=None):
             data=serialized_redemption,
             message=mes
         ), 200)
+
+    @classmethod
+    @roles_required(["finance"])
+    def get(cls, redeem_id=None):
```

Comment:
Why has this been created? Why not add the finance role to the already existing GET request?

Reply:
Good point @Andretalik

Reply:
@solnsumei

Reply:
Will revert and commit, thanks for the feedback. Had the notion that there should be finance only endpoint because of the put resource already existing. Thanks for the clarification.